ocaml/idl/datamodel.ml

```diff
@@ -571,6 +571,8 @@ let _ =
     ~doc:"Failed to disable SR-IOV on PIF" ();
   error Api_errors.network_has_incompatible_sriov_pifs ["PIF"; "network"]
     ~doc:"The PIF is not compatible with the selected SR-IOV network" ();
+  error Api_errors.vlan_network_incompatible ["PIF"; "network"]
```

Comment:
Below we call the error `network_incompatible_purposes`, why not using a similar same style here as well and call this `network_incompatible_with_vlan`. Or `cannot_add_vlan_to_network` like the errors further down. We are already inconsistent enough, I think we should try to stick to some styles

Reply:
Yes, I think `network_incompatible_with_X` is better.